pkg/chart/chart_test.go

```diff
@@ -78,3 +78,13 @@ func TestSaveChartNoRawData(t *testing.T) {
 
 	is.Equal([]*File(nil), res.Raw)
 }
+
+func TestName(t *testing.T) {
```

Comment:
Should this be renamed TestMetadata? Then the test would check the rest of the properties of metadata.
I am not sure of the value of just checking the name only.